test/credit_card_sanitizer_test.rb

```diff
@@ -102,6 +102,7 @@ class CreditCardSanitizerTest < MiniTest::Test
         assert_nil @sanitizer.sanitize!("blah blah  +4111111111111111.json")
         assert_nil @sanitizer.sanitize!("\"+4111111111111111\"")
         assert_nil @sanitizer.sanitize!("(+4111111111111111)")
+        assert_nil @sanitizer.sanitize!("&#43;1 936 321 1111")
```

Comment:
I don't think this number (`1 936 321 1111`) would get sanitized anyway; it doesn't appear to pass Luhn check and match our recognized credit card patterns.

Maybe just use
```
assert_nil @sanitizer.sanitize!("\"&#43;4111111111111111\"")
```